docs/deployment/overview.md

```diff
@@ -514,6 +514,10 @@ For example, our SaaS value looks like this (but with our Client ID redacted)
 }
 ```
 
+Should look like this:
```

Comment:
As per my other comment, I'd like to keep the instructions / screenshots in the documentation under the Authentication header. Let's remove this screenshot from here and keep it nice and clean.